var/spack/repos/builtin/packages/dyninst/package.py

```diff
@@ -95,21 +69,16 @@ class Dyninst(CMakePackage):
         conflicts("^intel-oneapi-tbb@2021.1:")
         conflicts("^intel-parallel-studio")
 
-    depends_on("intel-tbb@2019.9:", when="@13.0.0:")
-    depends_on("tbb@2018.6.0:", when="@10.0.0:12.3.0")
+    depends_on("tbb")
+    requires("^[virtuals=tbb] intel-tbb@2019.9:", when="@13.0.0:")
```

Comment:
I think we build with more than intel-tbb now, but I'd have to double-check. This is fine for now.